### ISSUE TITLE
Snackbar height on iPhones with notch

### ISSUE THREAD

**Issue by diego-lipinski-de-castro** [jonataslaw/getx#248]
Hello, I was testing Get snackbar layouts and as you can see the height of the snackbar is kinda off. It has some kind of margin or padding top that should not be there I guess? I dont understand what that blank space is nor why it is there as there is nothing that occupy that space.

get: ^2.13.1

```dart
Get.snackbar(
      "Erro ao adicionar refeição",
      "Verifique as informações e tente novamente",
      duration: Duration(seconds: 5),
      margin: EdgeInsets.zero,
      colorText: Colors.white,
      snackPosition: SnackPosition.BOTTOM,
      snackStyle: SnackStyle.GROUNDED,
      backgroundColor: Colors.grey[900],
    );
```

![image](https://user-images.githubusercontent.com/29608698/84777786-6e718c80-afb8-11ea-9c08-6396454a26cb.png)

**Comment by diego-lipinski-de-castro** [comment]
Just as a note, if snackPosition is TOP, it has the same weird behavior

![image](https://user-images.githubusercontent.com/29608698/84778228-112a0b00-afb9-11ea-9117-cc16fb8740ce.png)

**Comment by jonataslaw** [comment]
I wouldn't use the `snackStyle: SnackStyle.GROUNDED,` with iPhones.
Apple has already refused applications from me for using snackbars which, according to them, do not respect the human overlay guidelines.
I did Get.snackbar following overlapping patterns, with blur and semi transparency, with a higher notification, and I would not change that. Android you can do whatever you want, but apple is boring with that. Perhaps in future updates I will remove this option from iOS, and throw an exception when used.
The edges are bigger than necessary due to the last update that prevented the opening of the keyboard from hiding the snackbar.
I will still wonder if there is a way to resolve this, without a regression.
But I would tell you not to even use Grounded, I was not very careful with this API for iOS because it shouldn't even be used on apple devices.

**Comment by diego-lipinski-de-castro** [comment]
Well that makes sense, I am guessing now what is the equivalent of a snackbar for iOS? Im not sure if the "notification style" snackbar will fit right for the purpose it serves, but if it is, it is. I will take a look at that tho. Feel free to close, I am not doing it myself because you may want to keep it open to track future possibilities of changing the behavior by `I will still wonder if there is a way to resolve this, without a regression.`

**Comment by jonataslaw** [comment]
I will still decide if I use GetPlatform.isIOS to generate an exception if Grounded is used on an iOS device, or if I remove the padding that is making the edges bigger on iOS without regression for those who want to use it at their own risk.
I'm going to think about it and decide whether to keep this issue open. Thanks.

**Comment by jonataslaw** [comment]
Fix on 3.4.1
